.github/workflows/my-ci-pipeline.yml

```
@@ -15,3 +15,13 @@ jobs:
     -  run: docker build --target test --tag my-test-image .
     -  run: docker run my-test-image
 
+  to-do-app-production:
+    name: Build and Deploy to Azure
+    runs-on: ubuntu-latest
+    needs: to-do-app-tests
```

Don't forget to add a check that we're only releasing on a fixed branch (defaulting to main) for this, otherwise you'll find you're releasing every commit on every branch, regardless of whether it's working, something like:
```suggestion
    needs: to-do-app-tests
    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
```

---

Thanks Jack. I've made that change too and it's working (well, it didn't deploy as it was still syncing to the branch of module_9 when I tested it. I will check again once I've merged the pull request to Main and sync it). Thank you for the help as always.